modules/ROOT/pages/fd-share-to-studio.adoc

```diff
@@ -1,4 +1,4 @@
-= Collaborate with Someone who Works in Studio
+= Collaborate with Someone who Works in Studio or Flow Designer
```

Comment:
```suggestion
= Collaborate with Someone Who Works in Studio or Flow Designer
```
Capitalization does not depend so much upon the length of the word as upon its function, EXCEPT in the case of prepositions and conjunctions.